x-pack/plugins/stack_alerts/public/rule_types/threshold/expression.tsx

```diff
@@ -288,7 +288,7 @@ export const IndexThresholdRuleTypeExpression: React.FunctionComponent<
       <EuiSpacer size="s" />
       <EuiFormRow
         helpText={i18n.translate('xpack.stackAlerts.threshold.ui.filterKQLHelpText', {
-          defaultMessage: 'Use a KQL expression to limit the scope of your alert trigger.',
+          defaultMessage: 'Use a KQL expression to limit the scope of your alerts.',
```

Comment:
>          defaultMessage: 'Use a KQL expression to limit the scope of your alerts.',

I edited this at the same time, since "alert trigger" isn't terminology we use AFAIK. If this new text isn't accurate, I'm happy to iterate further.

Reply:
I think the change is accurate and makes sense